package.json

```diff
@@ -19,6 +19,7 @@
     "ajv": "6.12.3",
     "clone": "^2.1.2",
     "content-type": "^1.0.4",
+    "css-unit-converter": "^1.1.2",
```

Comment:
Best library I could find for easy sizing unit conversions (needed because exiftool gives out sizes with units, which are not always necessarily pixels!).